fortex/health/processors/scispacy_processor.py

```diff
@@ -93,36 +93,24 @@ def _process(self, input_pack: DataPack):
             else:
```

Comment:
we should have a check for self.configs.pipe_name == "hyponym_detector" too.
and through an error message if its not from the options in config.